fetch.bs

```diff
@@ -7091,22 +7070,26 @@ descendants share a quota of 384 kibibytes. That value is computed as such:
  <li><p>Let <var>quotaForRequestOrigin</var> be 64 kibibytes.
 
  <li>
-  <p><a for=list>For each</a> <a>deferred fetch record</a> <var>deferredRecord</var> of
-  <var>controlDocument</var>'s <a>fetch group</a>'s
-  <a for="fetch group">deferred fetch records</a>:</p>
+  <p><a for=list>For each</a> <var>navigable</var> in <var>controlDocument</var>'s
+  <a>node navigable</a>'s <a>descendant navigables</a> whose <a for=navigable>active document</a>'s
+  <a>deferred-fetch control document</a> is <var>controlDocument</var>:
 
   <ol>
-   <li><p>If <var>deferredRecord</var>'s <a for="deferred fetch record">invoke state</a> is
-   "<code>done</code>" or "<code>aborted</code>", then <a for=iteration>continue</a>.
+   <li>
+    <p><a for=list>For each</a> <a>deferred fetch record</a> <var>deferredRecord</var> of
```

Comment:
> For each deferred fetch record deferredRecord of controlDocument’s fetch group’s deferred fetch records:

Should this for-loop be put here? It relies only on `controlDocument` and has no relationship with `navigable`.

Reply:
It should work on the navigable's active document, fixed